pom.xml

```diff
@@ -293,6 +294,12 @@
             <groupId>org.apache.spark</groupId>
             <artifactId>spark-sql_${scala.major.version}</artifactId>
             <version>${spark.version}</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <version>${slf4j-api.version}</version>
 			<scope>provided</scope>
```

Comment:
@gkorland , we have slf4j provided by Spark

Reply:
This looks to be redundant since comes as a transitive dependency from spark-core. Fixed.